alerter.c

```diff
@@ -20,6 +23,7 @@ void alertInCelcius(float farenheit) {
         // However, this code doesn't count failures!
         // Add a test below to catch this bug. Alter the stub above, if needed.
         alertFailureCount += 0;
+        assert(alertFailureCount!=0);

```

Comment:
The test code (assert) is in the production code itself. Separate the tests from the production - once you prove your code, it doesn't need the tests to run in production.